stdlib/ballerina-http/src/main/ballerina/http/ws/websocket_connector.bal

```diff
@@ -61,15 +61,16 @@ public type WebSocketConnector object {
 
         P{{statusCode}} Status code for closing the connection
         P{{reason}} Reason for closing the connection
-        P{{timeoutInSecs}} timeout which waits for the close frame from remote endpoint to close the connection.
-                           If the timeout exceeds then the connection will be terminated even though a close frame
-                           is not received from the remote backend. If the value is -1 the connection will wait until
-                           a close frame is received.
-                           Default value 0: After sending the close frame connection will be closed automatically
-                           (Timeout is 0).
+        P{{timeoutInSecs}} Timeout which waits for the close frame from remote endpoint before closing the connection.
+                           If the timeout exceeds then the connection is terminated even though a close frame
+                           is not received from the remote endpoint. If the value < 0 (eg: -1) the connection waits
+                           until a close frame is received.
+                           Default value 60: Connection waits 60 seconds before closing the underlying TCP channel.
+                           If WebSocket frame is received from the remote backend within 60 seconds the connection is
+                           terminated immediately.
```

Comment:
Shall we rephrase the first sentence? Unclear what it says.

Also, do we need to explicitly document the default values? Doesn't Docerina automatically grab those from the function params?

Reply:
Oh Yes. Will make the changes :)